app/models/archive.coffee

```diff
@@ -54,14 +54,14 @@ class Archive extends Spine.Model
   # progress_strict calculates progress using raw complete and total values from the API
   progress_strict: =>
     unless @stats? then return 0
-    Math.floor(@stats.complete / (@stats.total) * 100)
+    Math.floor(@recordsComplete() / (@stats.total) * 100)
```

Comment:
I don't think this will work because `formatNumber` in `recordsComplete` will return something like `1,234,456`, not a number.